config/samples/infrastructure_v1beta1_jujucluster.yaml

```diff
@@ -10,8 +10,39 @@ metadata:
   name: jujucluster-sample
 spec:
   # using loadbalancer requires metallb on vsphere
+  model:
+    name: jujucluster-sample
+    cloud: jujucluster-sample 
+    cloudRegion: Boston
+    credentialName: jujucluster-sample
```

Comment:
Given that:
1. The only valid value for `spec.model.cloud` is that it must match `spec.cloud.name` (because it's the only cloud defined), and
2. The only valid value for `spec.model.credentialName` is that it must match whatever name is specified in the credentials secret data.value

It seems to me like these two fields provide no value and could be removed to save users some trouble.

Reply:
I had the same thoughts initially butI think there was some reason I added those, lemme look through things

Reply:
removed these fields and used the name from the cloud portion of the spec, and rely on the name provided in the credential secret.

Reply:
I also removed the remaining uses of jujuCluster.Name where required. Most of the uses are still valid, but there were a couple that were indeed incorrect